alephium/lib/wormhole.ts

```diff
@@ -27,13 +22,13 @@ interface NetworkConfigs {
 
 const networkConfigs: Record<NetworkType, NetworkConfigs> = {
     "mainnet": {
-        minimalConsistencyLevel: MAINNET_ALPH_MINIMAL_CONSISTENCY_LEVEL
```

Comment:
Why do we make this change? Isn't it good to reuse the constants?

Reply:
Because we have not released the SDK, CI can't download the dependency, I plan to release the SDK after the contract is finalized, and I'll update this once we release the SDK. WDYT?

Reply:
agreed